resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -342,6 +343,10 @@ public Builder writableStackTraceEnabled(boolean writableStackTraceEnabled) {
          * Configures an interval function with a fixed wait duration which controls how long the
          * CircuitBreaker should stay open, before it switches to half open. Default value is 60
          * seconds.
+         * <p>
+         * <b>ATTENTION!!!</b> Do not use with {@link #waitIntervalFunctionInOpenState(IntervalFunction)}!
```

Comment:
This <b>ATTENTION!!!</b> is too strong :) 
I would remove it or replace with more formal "Be aware that ..."